desktop/src/common/constants/constants.ts

```diff
@@ -109,7 +109,7 @@ export const ApiVersion = {
     },
     commerce: "2016-08-31-preview",
     authorization: "2017-05-01",
-    aadGraph: "1.6",
+    aadGraph: "1.6", // Deprecated
```

Comment:
Can we just remove all the AAD Graph code (and any related dependencies) instead of deprecating?

Reply:
Sure! No actual reason to keep the code around.